src/pages/restaurant/components/menu/helpers/getKeysOfAnObject.js

```diff
@@ -0,0 +1,8 @@
+//Helpers
```

Comment:
i think this is redundant helper because you can use lodash _keys method directly